src/actions/get-sbtc-bridge-config.ts

```diff
@@ -0,0 +1,25 @@
+"use server";
+
+import { cache } from "react";
+
+export default cache(async function getSbtcBridgeConfig() {
```

Comment:
Just to confirm, these are the env variables that will be forwarded to the client, correct?

Reply:
yes